### ISSUE TITLE
Allow Users to Override Styles

### ISSUE THREAD

**Issue by Spectere** [Spectere/Threads#13]
Users should be allowed to quickly and easily override story styles and define their own. This can be useful for accessibility and clashes of personal taste.
